src/pymor/analyticalproblems/expressions.py

```diff
@@ -475,48 +544,93 @@ def numpy_expr(self):
         return (f'(lambda _a: {self.numpy_symbol}(_a.reshape(_a.shape[:-{len(self.arg.shape)}] + (-1,)), '
                 f'axis=-1))({self.arg.numpy_expr()})')
 
+    def fenics_expr(self, params):
+        import ufl
+        if self.fenics_symbol is None:
+            raise NotImplementedError(f'UFL does not support operand {self.numpy_symbol}')
+        ufl_op = getattr(ufl, self.fenics_symbol)
+        return np.vectorize(ufl_op)(self.arg.fenics_expr(params))
```

Comment:
I think this code hasn't been tested yet, right? At least as subclasses seem to set `fenics_symbol` to `None`. I think at least having `min`, `max`, `norm` would be good. Ping me if you don't now how to handle these.

Reply:
I'll look into this the coming weeks!